### ISSUE TITLE
Feature request: Suspended Tabs

### ISSUE THREAD

**Issue by chuckharmston** [bwinton/SnoozeTabs#208]
This is all very personal and anecdotal, but I've found that time-based tab snoozing isn't very useful for me. I don't know when want a tab to come back, but I do know that I want it to go away for now. Bookmarks _sort of_ solve this need, but it also perfectly fits the goal of something like Snooze Tabs, and would be a natural inclusion into something like this experiment.

Proposal: allow tabs to be indefinitely suspended, to appear in a sort of filterable inbox for later retrieval. Happy to help consider potential UIs, but @bwinton is better at that sort of thing than I. 😄

**Comment by bwinton** [comment]
I'm interested in how bookmarks are lacking for this sort of thing…  Could you explain it a little more?

**Comment by tomprince** [comment]
This is exactly what I use [pocket](https://getpocket.com/) for.

**Comment by fishzle** [comment]
I have lots of bookmarks.
I have a mental model that some are more important than others. Eg,

1. read up on tax changes before end of financial year!!!!
2. plan for black Friday sale items!!

But the bookmark view doesn't give me a sense of urgency. I have tried using bookmark folders with time "urgent", "a bit later", "a lot later". But the maintenance is rubbish.

Especially when I'm lazy at that too.

I tried with bookmark labels, but I found that I ignored them because they weren't strong enough.

adding a "time component" to bookmarks is a bit like putting order into the set of bookmarks. Particularly for those bookmarks that are related to "tasks" and not "remember me, I've got nice pictures of cats".
